packages/dcos-metrics/extra/metrics.json

```diff
@@ -16,7 +16,7 @@
             {"key": "output_collector_chunk_size_datapoints", "value": "100"},
             {"key": "output_collector_chunk_timeout_seconds", "value": "10"},
             {"key": "state_path_dir",
-             "value": "/var/run/mesos/isolators/com_mesosphere_MetricsIsolatorModule/"}
+             "value": "/run/dcos/mesos/isolators/com_mesosphere_MetricsIsolatorModule/"}
```

Comment:
Question.

Don't have to create this path before using?
I see at-least metronome doing this ( https://github.com/dcos/dcos/blob/master/packages/metronome/build#L27)